iphone/Classes/KrollContext.m

```diff
@@ -147,17 +147,8 @@ TiValueRef ThrowException(TiContextRef ctx, NSString *message, TiValueRef *excep
 
 static TiValueRef MakeTimer(TiContextRef context, TiObjectRef jsFunction, TiValueRef fnRef, TiObjectRef jsThis, TiValueRef durationRef, BOOL onetime)
 {
-  static dispatch_once_t timerInitializer;
```

Comment:
Are you certain there's no thread contention here any more?

Reply:
This seems to work fine so far (git-diffing where the initial locking came from):
```js
var win = Ti.UI.createWindow({
    backgroundColor: '#fff'
});

var btn = Ti.UI.createButton({
    title: 'Trigger'
});

btn.addEventListener('click', function() {
    setTimeout(function () {
        Ti.API.info('1');
        setTimeout(function() {
            Ti.API.info('LAST');
        }, 500);
    }, 50);
    setTimeout(function () {
        Ti.API.info('2');
    }, 100);
    setTimeout(function () {
        Ti.API.info('3');
    }, 150);
    setTimeout(function () {
        Ti.API.info('4');
    }, 200);
    setTimeout(function () {
        Ti.API.info('5');
   }, 250);
});

win.add(btn);
win.open();
```